packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -143,11 +147,12 @@ class ShareVmButton extends Component {
 
   render () {
     return (
-      <TabButton
+      <ActionButton
```

Comment:
Thinking material: (not necessarily requesting a change)
Do we really need a `ShareVmButton` component?
Do we need to subscribe to resource sets (ie fetching all the resource sets every 5 secs) when we only use them once when the user clicks the button?
We could simply pass the resource set ID to `shareVm` and when we show the confirmation modal, we could call `resourceSet.getAll` only once (or even `resourceSet.get`) to get the resource set's info.